storage/src/tests/persistence/persistencetestutils.cpp

```diff
@@ -76,15 +70,14 @@ PersistenceTestUtils::PersistenceTestUtils() = default;
 PersistenceTestUtils::~PersistenceTestUtils() = default;
 
 std::string
-PersistenceTestUtils::dumpBucket(const document::BucketId& bid, uint16_t disk) {
-    assert(disk == 0u);
+PersistenceTestUtils::dumpBucket(const document::BucketId& bid) {
     return dynamic_cast<spi::dummy::DummyPersistence&>(_env->_node.getPersistenceProvider()).dumpBucket(makeSpiBucket(bid));
 }
 
 void
-PersistenceTestUtils::setupDisks(uint32_t numDisks) {
-    _env = std::make_unique<PersistenceTestEnvironment>(DiskCount(numDisks), "todo-make-unique-persistencetestutils");
-    setupExecutor(numDisks);
+PersistenceTestUtils::setupDisks() {
+    _env = std::make_unique<PersistenceTestEnvironment>("todo-make-unique-persistencetestutils");
+    setupExecutor(2);
```

Comment:
Should this be 1 instead?

Reply:
Yes it should, I just tested with 2 too as it did not see the connection between disk and executors.